input/definitions/MeasureReport/StructureDefinition-criteriaReference.json

```diff
@@ -0,0 +1,88 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "criteriaReference",
+  "url": "http://hl7.org/fhir/StructureDefinition/criteriaReference",
```

Comment:
Per convention, because this is an extension for MeasureReport specifically, the url should probably be:
http://hl7.org/fhir/StructureDefinition/measurereport-criteriaReference

Reply:
```suggestion
  "url": "http://hl7.org/fhir/StructureDefinition/cqf-criteriaReference",
```